craco.config.cjs

```diff
@@ -81,6 +97,15 @@ module.exports = {
             })
           }
 
+          // Increase the cacheable file size (by the ServiceWorker) so that main.<hash>.js is cached.
+          if (plugin instanceof InjectManifest) {
```

Comment:
Is this a feature (separate PR) or something to provide Webpack 4-like functionality due to change in cacheable file size as of Webpack 5?

Reply:
This is a required part of the migration.
Webpack 4 split the entry file into entry (very small) and main (initial assets), but Webpack 5 optimizes by serving it all as a single file. As a result, that file is much larger, and so we need to bump the limit on cacheable file size by the workbox InjectManifest plugin.

Reply:
The CRA / Webpack 5 configuration no longer splits the entry chunk, so the entry chunk (main.js) is now much larger. Instead of allowing a larger cached file, I'm going to make a vendor bundle to better mimic the Webpack 4 behavior and for better performance.